doc/6/getting-started/react/with-redux/index.md

```diff
@@ -0,0 +1,171 @@
+---
+type: page
+code: false
+title: React
+description: Getting started with Kuzzle and React with Redux
+order: 0
+---
+
+
+# Getting Started with Kuzzle and React with Redux Saga
+
+This section deals with **Kuzzle** (+ **Javascript SDK**) and **React** (with **Redux** and **Redux Saga**). We will create **documents** in Kuzzle and subscribe to [document notifications](/sdk/js/6/essentials/realtime-notifications/#document-messages) to develop a realtime chat.
+
+## Requirements
+
+- **Node.js** >= 8.0.0 ([install here](https://nodejs.org/en/download/))
+- **Create React App** ([install here](https://github.com/facebook/create-react-app))
+- **Running Kuzzle Stack** ([instructions here](core/1/guides/getting-started/running-kuzzle/))
+
+## Prepare your environment
+
+Create your React app and install all the dependencies from the command line using yarn:
+
+```bash
+yarn create react-app kuzzle-playground
+cd kuzzle-playground
+yarn add kuzzle-sdk redux redux-saga react-redux
+```
+
+We'll rewrite the _src/App.js_ so you can remove everything inside.
+
+## Instanciating Kuzzle SDK
+
+We have to connect the server so that our client can interact with it.
+
+To do this, we have to create _src/services/kuzzle.js_ file to put our kuzzle instance, a bit like a singleton:
+
+<<< ./snippets/kuzzle.js.snippet[js]
+
+You can now edit the _src/App.js_ file to connect to Kuzzle.
+To do this, import the kuzzle service file:
+
+<<< ./snippets/App.js.snippet:1[js]
+
+Add the following imports in the same time:
+
+<<< ./snippets/App.js.snippet:5[js]
+
+Now, add the `app` class and add in the constructor a `message` property in the state (we'll use it to store the user input message) and a call to an `_initialize()` function: 
+
+<<< ./snippets/App.js.snippet:6[js]
+
+After that, create that function with the connection to Kuzzle:
+
+<<< ./snippets/App.js.snippet:2[js]
+
+Then we will [etablish the connection](/sdk/js/6/core-classes/kuzzle/connect/) to kuzzle and create, if they don't [exists](sdk/js/6/controllers/index/exists/), the [index](sdk/js/6/controllers/index/create/) and [collection](sdk/js/6/controllers/collection/create/) of our chat. 
```

Comment:
```suggestion
Then we will [establish the connection](/sdk/js/6/core-classes/kuzzle/connect/) to kuzzle and create, if they don't [exist](sdk/js/6/controllers/index/exists/), the [index](sdk/js/6/controllers/index/create/) and [collection](sdk/js/6/controllers/collection/create/) for our chat. 
```

Reply:
There is the same sentence [in the VueJS getting started](https://github.com/kuzzleio/sdk-javascript/pull/431#discussion_r311571456), but you haven't asked for the same changes. I need to change it there too ?

Reply:
Ah, yes, I didn't saw the typo on `establish` on this other PR. I fixed my suggestion there.